packages/python-packages/api-stub-generator/apistub/_apiview.py

```diff
@@ -7,12 +7,18 @@
 from ._token import Token
 from ._token_kind import TokenKind
 from ._version import VERSION
+from ._diagnostic import Diagnostic
 
-JSON_FIELDS = ["Name", "Version", "VersionString", "Navigation", "Tokens"]
+JSON_FIELDS = ["Name", "Version", "VersionString", "Navigation", "Tokens", "Diagnostics"]
 
 HEADER_TEXT = "# Package is parsed using api-stub-generator(version:{})".format(VERSION)
 COMPLEX_DATA_TYPE_PATTERN = re.compile("([a-zA-Z]+)(\[|\()[^\]\)]+(\]|\))")
 
+# Lint warnings
+SOURCE_LINK_NOT_AVAILABLE = "Source definition link is not available for [{0}]. Please check and ensure type is fully qualified name in docstring"
+RETURN_TYPE_MISMATCH = "Return type in type hint is not matching return type in docstring"
```

Comment:
I suspect we'll end up with quite a lot of these in future - so longer term we could look at moving these out into their own file.